datasets/turk/README.md

```diff
@@ -0,0 +1,142 @@
+---
+title: 'TURKCorpus'
+type: Simplification
+motivation: TURKCorpus is a high-quality simplification dataset where each source sentence is associated with 8 human-written simplifications.
```

Comment:
Can you provide valid dataset tags here ?
You can find more information here:
https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card